lib/systemd/system/revpi-hciuart.target

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Revolution Pi Bluetooth Device
+
+# support max 4 similar devices
+Wants=revpi-hciuart@ttyUSB0.service
```

Comment:
I didn't find a good way to "monitor" those devices and trigger the service.
Just find a similar one "serial-getty@.service", which is actually in this way --- enumerate from tty1 to tty6/7